src/Avalonia.Diagnostics/Diagnostics/VisualTreeDebug.cs

```diff
@@ -10,7 +10,7 @@ public static class VisualTreeDebug
     {
         public static string PrintVisualTree(IVisual visual)
         {
-            StringBuilder result = new StringBuilder();
+            var result = new StringBuilder();
```

Comment:
Why not `StringBuilderCache` here?

Reply:
because I didn't want to add `<InternalsVisibleTo Include="Avalonia.Diagnostics, PublicKey=$(AvaloniaPublicKey)"/>` to Avalonia.Base